New_Home_Work_2_task_1

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+/*
+ * Сжать массив, удалив из него все нули. Освободившиеся место справа заполнить -1
+ */
+namespace task_2_1
+{
+    class Program
+    {
+        /// <summary>
+        /// Find zero elements in the array and reduce the dimension of the array.
+        /// </summary>
+        /// <param Array of the elements ="Array">Значение</param>      
+        /// <param beginning of the array ="n">Значение</param>      
+        /// <param full size of the array ="s">Значение</param>
```

Comment:
Main method has no Array, n, s like arguments.

Reply:
done it